umbra-js/src/classes/Umbra.ts

```diff
@@ -8,7 +8,6 @@ import {
   BigNumber,
   BigNumberish,
   Contract,
```

Comment:
*Arbitrary comment location*

There should be an "x" on each group of send fields after the first one that allows the user to remove it from the list.

Reply:
Added here: [335d2b8](https://github.com/ScopeLift/umbra-protocol/pull/491/commits/335d2b857b8fa898c365c1288aca95a4877131b4)

Reply:
Nice! Looking good and working well. One improvement we need: it shouldn't be possible to delete the last row.

![delete-last-row](https://user-images.githubusercontent.com/1479802/235518251-113b32d7-7ad2-4b6b-b658-3c1b73a89027.gif)

Reply:
Fixed in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/d51628fb4803b9a2f67a3c8cd373760d2069234b

Reply:
[Screencast from 2023-05-04 11-10-10.webm](https://user-images.githubusercontent.com/61768337/236250568-2fcfa479-9bdd-4a8f-9ebf-9fc599566a4b.webm)